graph/nullable.md

```diff
@@ -0,0 +1,312 @@
+# Nullable Properties
```

Comment:
@OlgaPodo is this going to be standard practice now. to have these guidance expansion topics vs pattern topics?

Reply:
I did this as a bit of a "workaround" since explaining the nuances of nullability and required and such aren't really a "pattern". I don't think this should be standard practice per se, but I do think it's important to not call "guidelines" patterns. I'm open to restructuring wherever makes the most sense.